### DISCUSSION TITLE
Copying project

### DISCUSSION THREAD

**Discussion by kicaj** [community/community#61842] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

During the copying project I always receive error "There was a problem copying the project. Configuration 'xAxis.groupBy.column' must be an integer", but project is copied without any issues/tasks.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by BaraaBasim** [comment]
Can you possibly provide more details? what type of project and the dependencies?

**Comment by kicaj** [comment]
Hmm...
- project is private
- has custom field of type "single select"
- has custom field of type "iteration"
- just one user as admin
- has some filters
- has serval tasks not issues
That's all.

**Reply by braianj** [reply]
anything here?

**Comment by ifeltsweet** [comment]
We had a similar problem. To fix, navigate to project Insights and check all the charts. Some of the charts will have invalid configuration (the UI will tell you this). After you fix them you should be able to copy your project.
